test/MetagenomeAPI_server_test.py

```diff
@@ -153,6 +153,22 @@ def save_metagenome(self):
         })[0]
         return '/'.join([str(obj_info[6]), str(obj_info[0]), str(obj_info[4])])
 
+    # @unittest.skip('x')
+    def test_get_contig_info(self):
+        """test the 'get_contig_info' function
+        NOTE: This test is tied to a version of workspace object in elasticsearch.
+        """
+        ref = "43655/58/1"
+        params = {
+            "ref": ref,
+            "contig_id": "Ga0065724_100164"
+        }
+        ret = self.getImpl().get_contig_info(self.getContext(), params)[0]
+        self.assertTrue('contig' in ret)
+        self.assertTrue('length' in ret['contig'])
```

Comment:
Can you actually assertEqual on the values here? Did you check to make sure the values are correct?